OptimizelySDK/Utils/DateTimeExtensions.cs

```diff
@@ -0,0 +1,33 @@
+﻿/* 
+ * Copyright 2019, Optimizely
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+
+namespace OptimizelySDK.Utils
+{
+    public static class DateTimeExtensions
```

Comment:
We already have [DateTimeUtils ](https://github.com/optimizely/csharp-sdk/blob/master/OptimizelySDK/Utils/DateTimeUtils.cs) cIass, I think we can use that instead of adding one more Datetime util class. So we can either remove that or we can move these functions into that class.